deny.toml

```diff
@@ -19,7 +19,7 @@ vulnerability = "deny"
 # The lint level for unmaintained crates
 unmaintained = "deny"
 # The lint level for crates that have been yanked from their source registry
-yanked = "deny"
+yanked = "warn"
```

Comment:
Downgrading, but this is the default value IIRC.

Either way, `crossbeam-channel` yanked a version of their crate, which is a dependency of `criterion`, so we can't do anything about it until they upgrade.